src/burn/engine/exeengine.cpp

```diff
@@ -165,6 +115,146 @@ extern "C" HRESULT ExeEngineParsePackageFromXml(
     return hr;
 }
 
+static HRESULT ParseCommandLineArgumentsFromXml(
+    __in IXMLDOMNode* pixnExePackage,
+    __in BURN_PACKAGE* pPackage
+    )
+{
+    HRESULT hr = S_OK;
+    IXMLDOMNodeList* pixnNodes = NULL;
+    IXMLDOMNode* pixnNode = NULL;
+    DWORD cNodes = 0;
+    LPWSTR scz = NULL;
+
+    // select exit code nodes
```

Comment:
The WiX style is for comments to be complete sentences, so they should always be capitalized and have periods.